src/app/app.component.ts

```diff
@@ -7,7 +8,7 @@ import { Component } from '@angular/core';
 })
 export class AppComponent {
   title = 'angular-http-crud1';
-  constructor() {}
+  constructor(public cartService: CartService) {}

```

Comment:
not recommended to be public.
could have made it private and created a function that does the count in .ts file